site/faq.md

```diff
@@ -251,6 +252,12 @@ Now restart fluxd to re-read the k8s secret (if it is running):
 
 `kubectl delete $(kubectl get pod -o name -l name=flux)`
 
+### Can I run Flux with readonly Git access?
+
+Yes.  You can use the `--git-readonly=true` command line argument.  The Helm chart exposes this as `git.readonly`.
```

Comment:
```suggestion
Yes. You can use the `--git-readonly=true` command line argument.  The Helm chart exposes this as `git.readonly`.
```

Reply:
apparently the internet thinks I'm [over 40 years old](https://www.cultofpedagogy.com/two-spaces-after-period/), haha.  I was taught to write on typewriters (underfunded small town school) and haven't seen a reason to change. heh.